### DISCUSSION TITLE
How to: Save your Private Messages on the old Forum

### DISCUSSION THREAD

**Discussion by jusjoken** [OpenSageTV/Community#21] [General]
If you would like to save a copy of your PM's on the old forum PRIOR to it being archived follow these instructions...

- select the User-CP menu item
- select List Messages
- at the bottom of the list click a download option under Download all Private Messages as:

Done

---
@jusjoken
If you wish to see what I am up to and support my efforts visit my [Patreon page](https://www.patreon.com/user?u=58035567)

**Comment by KryptoNyte-Sage** [comment]
Worked great!  Thank you.

**Comment by Sagetvmaster** [comment]
Bingo, thank you sir

**Comment by seansd** [comment]
I was so late to the party.  I had no idea that the move happened until literally this week.  I'm guessing there is no shot in getting back my old threads now. . .but figured I'd ask.
